torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -81,10 +93,33 @@ def _create_label(self, edge_label_index, edge_label):
 
         return edge_label_index, edge_label
 
+    def _modify_node_time(self, query_dict, edge_time):
+        def update_time(input_type):
+            index = query_dict[input_type]
+            new_node_time, _ = scatter_max(edge_time, index,
+                                           dim_size=self.num_src_nodes)
+            index_unique = index.unique()
+            self.node_time_dict[input_type][index_unique] = new_node_time[
+                index_unique]
+
+        update_time(self.input_type[0])
+        # TODO(jinu) input_type[0] = input_type[1]
```

Comment:
I don't mind that much, but the TODO doesn't make it very clear to me what is actually pending here.

Reply:
Will update that. When the `node_type`s we should take `max` across both `src` and `dst` edges.